docs/actions/create_app_online.md

```diff
@@ -305,7 +305,7 @@ Key | Description | Default
   `company_name` | The name of your company. Only required if it's the first app you create | 
   `skip_itc` | Skip the creation of the app on iTunes Connect | `false`
   `itc_users` | Array of iTunes Connect users. If provided, you can limit access to this newly created app for users with the App Manager, Developer, Marketer or Sales roles | 
-  `enabled_features` | [DEPRECATED!] Please use `enable_services` instead - Array with Spaceship App Services | `{}`
+  `enabled_features` | **DEPRECATED!** Please use `enable_services` instead - Array with Spaceship App Services | `{}`
   `enable_services` | Array with Spaceship App Services (e.g. app_group: (on|off), apple_pay: (on|off), associated_domains: (on|off), data_protection: (complete|unlessopen|untilfirstauth), game_center: (on|off), health_kit: (on|off), home_kit: (on|off), hotspot: (on|off), icloud: (legacy|cloudkit), in_app_purchase: (on|off), inter_app_audio: (on|off), multipath: (on|off), network_extension: (on|off), nfc_tag_reading: (on|off), personal_vpn: (on|off), passbook: (on|off), push_notification: (on|off), siri_kit: (on|off), vpn_configuration: (on|off), wallet: (on|off), wireless_accessory: (on|off)) | `{}`
```

Comment:
This is [broken](https://deploy-preview-608--fastlane-docs-preview.netlify.com/actions/create_app_online/#parameters_1).

Reply:
Same as https://github.com/fastlane/docs/pull/608#discussion_r189327396.